.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "develop", hotfix/*, master, release/*, feature/** ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "develop", hotfix/*, master, release/* ]
+  schedule:
+    - cron: "30 8 * * 3"
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'python', 'javascript' ]
```

Comment:
This doesn't seem super helpful since the repo is all bash and Dockerfiles, but it can't hurt?

Reply:
Probably, I added javascript because of the package.json, but it doesn't look like it scans it. 
It did find 15 lines of python, looks like https://github.com/dockstore/compose_setup/blob/develop/scripts/generateLogstashPathPattern.py